2022-01-09 18:22:21,806 INFO    MainThread:17712 [wandb_setup.py:_flush():71] setting env: {}
2022-01-09 18:22:21,806 INFO    MainThread:17712 [wandb_init.py:_log_setup():371] Logging user logs to /content/tbcnn/wandb/offline-run-20220109_182221-2o1v4aoj/logs/debug.log
2022-01-09 18:22:21,806 INFO    MainThread:17712 [wandb_init.py:_log_setup():372] Logging internal logs to /content/tbcnn/wandb/offline-run-20220109_182221-2o1v4aoj/logs/debug-internal.log
2022-01-09 18:22:21,806 INFO    MainThread:17712 [wandb_init.py:init():404] calling init triggers
2022-01-09 18:22:21,807 INFO    MainThread:17712 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-09 18:22:21,807 INFO    MainThread:17712 [wandb_init.py:init():460] starting backend
2022-01-09 18:22:21,807 INFO    MainThread:17712 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-09 18:22:21,825 INFO    MainThread:17712 [backend.py:ensure_launched():216] starting backend process...
2022-01-09 18:22:21,833 INFO    MainThread:17712 [backend.py:ensure_launched():222] started backend process with pid: 17722
2022-01-09 18:22:21,840 INFO    MainThread:17712 [wandb_init.py:init():469] backend started and connected
2022-01-09 18:22:21,845 INFO    MainThread:17712 [wandb_init.py:init():533] updated telemetry
2022-01-09 18:22:21,910 INFO    MainThread:17712 [wandb_init.py:init():606] starting run threads in backend
2022-01-09 18:22:25,731 INFO    MainThread:17712 [wandb_run.py:_console_start():1810] atexit reg
2022-01-09 18:22:25,731 INFO    MainThread:17712 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-09 18:22:25,732 INFO    MainThread:17712 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-09 18:22:25,734 INFO    MainThread:17712 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-09 18:22:25,734 INFO    MainThread:17712 [wandb_init.py:init():633] run started, returning control to user process
2022-01-09 18:23:31,062 INFO    MainThread:17712 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-01-09 18:23:31,064 INFO    MainThread:17712 [wandb_run.py:_restore():1752] restore
